apps/ui/src/helpers/intercom.ts

```diff
@@ -6,7 +6,7 @@ export function startIntercom() {
   const w: any = window;
   w.intercomSettings = {
     app_id: APP_ID,
-    custom_launcher_selector: '.intercom-launcher'
+    custom_launcher_selector: '.intercom-launcher hidden md:block'
```

Comment:
When ever we navigate to different routes on snapshot.box
![image](https://github.com/user-attachments/assets/698ad5bb-b48d-4a49-965d-5dadac643e12)